### PR TITLE
Header images

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

**Comment by tiagojsag** [comment]
@andresgnlez can you finish up here please? thanks

**Comment by andresgnlez** [comment]
@tiagojsag done. Are images optimized?

**Comment by tiagojsag** [comment]
yes, but you might want to make them lower res if they are still too big

File: app/styles/_settings.scss [hunk 1]
```diff
@@ -17,6 +17,7 @@ $wrap-desktop: 115px;
 // Font sizes
 $font-size-tiny: 10px;
 $font-size-x-small: 12px;
+$font-size-xs-small: 13px;
```
Comment (tiagojsag) [inline app/styles/_settings.scss]: honest question: can't we just agree on a standard set of sizes? otherwise this loses total sense

Comment (andresgnlez) [inline app/styles/_settings.scss]: Give us a couple of days and we'll do a refactor..  😁  Anyway, that size is the UI Kit and spoken with Dani.

**Pull request merged**